From 6908e8eedf5232519a1f85942249040051e48fa8 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Timo=20Ter=C3=A4s?= <timo.teras@iki.fi>
Date: Wed, 30 May 2012 16:23:50 +0300
Subject: [PATCH 1/2] uclibc/res: do not call res_init

it prevents rereading of resolv.conf in uclibc, but enables that
for glibc.
---
 nameserv.c |    1 -
 1 file changed, 1 deletion(-)

diff --git a/nameserv.c b/nameserv.c
index 1770845..9ecf1d9 100644
--- a/nameserv.c
+++ b/nameserv.c
@@ -177,7 +177,6 @@ DNS_IPAddress2Name(IPAddr *ip_addr, char *name, int len)
 void
 DNS_Reload(void)
 {
-  res_init();
 }
 
 /* ================================================== */
-- 
1.7.10.2

